packages/edit-site/src/components/editor/index.js

```diff
@@ -144,8 +144,12 @@ export default function Editor( { listViewToggleElement, isLoading } ) {
 		: __( 'Block Library' );
 	const blockContext = useMemo( () => {
 		const { postType, postId, ...nonPostFields } = context ?? {};
+
 		return {
 			...( hasPageContentFocus ? context : nonPostFields ),
+			// Ideally this context should be removed. However, it is currently used by the Query Loop block.
+			templateSlug:
+				editedPostType === 'wp_template' ? editedPost.slug : undefined,
```

Comment:
```suggestion
				editedPostType === TEMPLATE_POST_TYPE ? editedPost.slug : undefined,
```

Not a huge deal, but it looks like we're already importing `TEMPLATE_POST_TYPE` above

Reply:
I understand that there might be different opinions here. I've found these constants to be unnecessary redirections (need to go to other files to know what these are). Do you find real value in these? 

I'm happy to use them btw.

Reply:
> Do you find real value in these?

I could go either way. 

The con: it's a bit of extra work to look up the constant. 

On the fence: they don't provide much more value than a global find and replace would serve, should the values ever change. Maybe a couple of seconds of time assuming that the constants are used consistently.

The pro: reading the constant I know that `wp_template` is a post type. `wp_template` on it's own, unless you're intimate with domain-specific knowledge, could be anything.